### PR TITLE
add timeline chart guide

### PR DESCRIPTION
**Pull request by SyafiqMuizz**

### CODE REVIEW

File: visual/timeline-chart.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Time Line
+
+![TimeLine](./images/timeline-chart/timeine.png)
```
Comment (tchyong) [inline visual/timeline-chart.md]: @SyafiqMuizz Since your images extension is PNG in big case. Please change all ``png`` to ``PNG``

Comment (SyafiqMuizz) [reply visual/timeline-chart.md]: okay, will edit the md file and recommit

File: visual/timeline-chart.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+![US President Death](./images/timeline-chart/output-1.PNG)
+
+**Binding**
+
+The diagram below shows the data binding for US President Deaths and Terms use case 
+
+![First Use Case Binding](./images/timeline-chart/binding-1.PNG)
+
+### Sample Use Case 2
+The FBI recorded the crimes that were happening within USA for the decade(1996-2016) to check whether the the crime rate would improve or worsen within the years to come. The timeline use case states the percentage of crime rate happening within the US against the total number of crimes that happen in chronological order.
```
Comment (tchyong) [inline visual/timeline-chart.md]: @SyafiqMuizz 
This use case not very suitable as timeline chart. This scenario end goal is to show the trend of the crime rate throughout years. Line chart would be more suitable as it show the trend and can quickly identify the rate result (increase/decrease) from the chart.

Comment (SyafiqMuizz) [reply visual/timeline-chart.md]: okay, will change this use case to one with a project management flow

**Pull request closed without merging**